content/providers/01-ai-sdk-providers/11-google-vertex.mdx

```diff
@@ -132,23 +132,47 @@ The following optional settings are available for Google Vertex models:
     - `BLOCK_ONLY_HIGH`
     - `BLOCK_NONE`
 
-### Example
+- **useSearchGrounding** _boolean_
+
+  Optional. When enabled, the model will [use Google search to ground the response](https://cloud.google.com/vertex-ai/generative-ai/docs/grounding/overview).
+
+### Example: generating text
 
 You can use Google Vertex language models to generate text with the `generateText` function:
 
-```ts
-import { vertex } from '@ai-sdk/google-vertex'
-import { generateText } from 'ai'
+```ts highlight="1,5"
```

Comment:
think this is highlighting the first and last line?

Reply:
first line and model line